src/components/Calculator.js

```diff
@@ -1,49 +1,65 @@
 import React from 'react';
-
 import Cell from './Cell';
+import calculate from '../logic/calculate';
 
 export default class Calculator extends React.Component {
   constructor(props) {
     super(props);
-    this.state = {};
+    this.state = {
+      calculateObj: {},
+      total: 0,
+    };
+  }
+
+  handleClick(e) {
+    console.log('>> ', this, e.target.firstChild.data);
```

Comment:
- [ ] It is not good practice to push console logs in the final iteration of your app. It kills performance and can expose confidential data in more sophisticated apps. Kindly remove it.